Make GHES support / setup more clear (#534)
actions/dependency-review-action: README.md

>  dependency-review-action

Reviewer: ```suggestion
 # dependency-review-action
```

Security](https://docs.github.com/en/enterprise-server@3.6/admin/code-security/managing-github-advanced-security-for-your-enterprise/enabling-github-advanced-security-for-your-enterprise)
and [GitHub
Connect](https://docs.github.com/en/enterprise-server@3.6/admin/github-actions/managing-access-to-actions-from-githubcom/enabling-automatic-access-to-githubcom-actions-using-github-connect)

> are enabled and that you sync the [dependency-review-action](https://github.com/actions/dependency-review-action) on to the server.

Reviewer: ```suggestion
are enabled, and that you have installed the [dependency-review-action](https://github.com/actions/dependency-review-action) on the server.
```
